.changeset/fair-terms-beg.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystone-6/core': patch
+'@keystone-6/fields-document': patch
+---
+
+Updates admin meta GraphQL schema to support singletons
```

Comment:
@mitchellhamilton will this be a separate patch to support singleton or will this go out together with singleton release as a major?

Reply:
This will go out in the same release